### PR TITLE
Set the role of the current cluster into the kubesphere-config configmap

### PR DESCRIPTION
**Pull request by iawia002**
Part of https://github.com/kubesphere/kubesphere/issues/4610

/cc @wansir

### CODE REVIEW

File: roles/ks-core/config/templates/kubesphere-config.yaml.j2 [hunk 1]
```diff
@@ -111,9 +111,14 @@ data:
...
+{% if multicluster.clusterRole is defined and multicluster.clusterRole == "member" %}
+    multicluster:
+      clusterRole: {{ multicluster.clusterRole }}
+{% endif %}
 {% if multicluster.clusterRole is defined and multicluster.clusterRole == "host" %}
     multicluster:
       enable: true
+      clusterRole: {{ multicluster.clusterRole }}
       agentImage: {{ tower_repo }}/{{ tower_image }}:{{ tower_image_tag }}
       proxyPublishService: tower.kubesphere-system.svc
```
Comment (wansir) [inline roles/ks-core/config/templates/kubesphere-config.yaml.j2:123]:
```suggestion
    multicluster:
      clusterRole: {{ multicluster.clusterRole }}
{% if multicluster.clusterRole is defined and multicluster.clusterRole == "member" %}
      enable: false
{% endif %}
{% if multicluster.clusterRole is defined and multicluster.clusterRole == "host" %}
      enable: true
      agentImage: {{ tower_repo }}/{{ tower_image }}:{{ tower_image_tag }}
      proxyPublishService: tower.kubesphere-system.svc
```

Comment (wansir) [inline roles/ks-core/config/templates/kubesphere-config.yaml.j2:123]:
```suggestion
    multicluster:
      clusterRole: {{ multicluster.clusterRole }}
{% if multicluster.clusterRole is defined and multicluster.clusterRole == "member" %}
      enable: false
{% endif %}
{% if multicluster.clusterRole is defined and multicluster.clusterRole == "host" %}
      enable: true
      agentImage: {{ tower_repo }}/{{ tower_image }}:{{ tower_image_tag }}
      proxyPublishService: tower.kubesphere-system.svc
```

Comment (iawia002) [reply roles/ks-core/config/templates/kubesphere-config.yaml.j2]: Updated, PTAL

**Pull request merged**